README.md

```diff
@@ -54,27 +63,34 @@ personal access token on your local OSF instance ([here](http://localhost:5000/s
 
 #### Create a local settings file
 
-To do this:
+If for some reason you don't have a config/local.yml you can generate one. To do this:
 ```bash
-ember g ember-osf-settings `echo $HOSTNAME`
+ember g ember-osf `echo $HOSTNAME`
```

Comment:
Now that we generate a file with a fixed filename, what is the purpose of the `HOSTNAME` argument?